### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'astNode')

### ISSUE THREAD

**Issue by valgog** [cjoudrey/graphql-schema-linter#323]
Hello, the linter is reporting the following error, so reporting it here :)

```
podman found, using it instead of docker
podman-compose not found, using docker-compose instead
[SCHEMA-LINTER] Linting GraphQL schema...
It looks like you may have hit a bug in graphql-schema-linter.

It would be super helpful if you could report this here: https://github.com/cjoudrey/graphql-schema-linter/issues/new

TypeError: Cannot read properties of undefined (reading 'astNode')
    at hasAllowedForAndDeprecated (/workspace/gql-linter/rules/directive-allowedfor.js:84:42)
    at Object.FieldDefinition (/workspace/gql-linter/rules/directive-allowedfor.js:69:11)
    at Object.enter (/workspace/node_modules/graphql-schema-linter/node_modules/graphql/language/visitor.js:323:29)
    at Object.enter (/workspace/node_modules/graphql-schema-linter/node_modules/graphql/utilities/TypeInfo.js:370:25)
    at visit (/workspace/node_modules/graphql-schema-linter/node_modules/graphql/language/visitor.js:243:26)
    at validate (/workspace/node_modules/graphql-schema-linter/node_modules/graphql/validation/validate.js:69:24)
    at validateSchemaDefinition (/workspace/node_modules/graphql-schema-linter/lib/validator.js:70:43)
    at run (/workspace/node_modules/graphql-schema-linter/lib/runner.js:68:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
make: *** [Makefile:391: schema-lint] Error 3

Please try running `git fetch origin && git merge origin/main` in your local branch and pushing.
```
